templates/telco/index.html

```diff
@@ -365,6 +368,9 @@ <h4>Accelerate your migration to NFV with Open Source MANO</h4>
       <p>
         <a href="/telco/osm">Read more&nbsp;&rsaquo;</a>
```

Comment:
```suggestion
        <a href="/engage/telco-network-functions-whitepaper">Read more&nbsp;&rsaquo;</a>
```

Reply:
We're adding a link to this block, on lines 371-373, the "Read more" link should still point to /telco/osm